_topic_map.yml

```diff
@@ -206,28 +206,31 @@ Topics:
 - Name: Installing on bare metal
   Dir: installing_bare_metal
   Topics:
-  - Name: Installing a cluster on bare metal
-    File: installing-bare-metal
-  - Name: Installing a cluster on bare metal with network customizations
-    File: installing-bare-metal-network-customizations
-  - Name: Restricted network bare metal installation
-    File: installing-restricted-networks-bare-metal
-- Name: Deploying installer-provisioned clusters on bare metal
-  Dir: installing_bare_metal_ipi
-  Distros: openshift-origin,openshift-enterprise
-  Topics:
-  - Name: Overview
-    File: ipi-install-overview
-  - Name: Prerequisites
-    File: ipi-install-prerequisites
-  - Name: Setting up the environment for an OpenShift installation
-    File: ipi-install-installation-workflow
-  - Name: Post-installation configuration
-    File: ipi-install-post-installation-configuration
-  - Name: Expanding the cluster
-    File: ipi-install-expanding-the-cluster
-  - Name: Troubleshooting
-    File: ipi-install-troubleshooting
+  - Name: Installing user-provisioned clusters on bare metal
+    Dir: installing_bare_metal_upi
+    Topics:
+    - Name: Installing a cluster on bare metal
+      File: installing-bare-metal
+    - Name: Installing a cluster on bare metal with network customizations
+      File: installing-bare-metal-network-customizations
+    - Name: Restricted network bare metal installation
+      File: installing-restricted-networks-bare-metal
+  - Name: Deploying installer-provisioned clusters on bare metal
```

Comment:
I'd s/Deploying/Installing to make it consistent with the rest of the installation topics.

Reply:
+1 - there was a bit of a confusion when we had the IPI and UPI based installers using different terminology for bare-metal.